### PR TITLE
Add Touche.app v1.1.1

### PR DESCRIPTION
**Pull request by artdevjs**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked that the cask was not already refused in [closed issues].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed

### CODE REVIEW

File: Casks/touche.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+cask 'touche' do
+  version '1.1.1'
+  sha256 '1430c9c7fdb2fe13981b2b182fd81f209e8bde971b6a220f64d47297072059ba'
+
+  # redsweater.s3.amazonaws.com was verified as official when first introduced to the cask
+  url "https://redsweater.s3.amazonaws.com/Touche#{version}.zip"
```
Comment (miccal) [inline Casks/touche.rb]: Can you please add an appcast:
```
appcast 'https://red-sweater.com/touche/appcast.php',
        checkpoint: '2221cef86f61ed01bdd44df1e8b2cd7052adec9dce235f845548b808d97843e7'
```

Comment (artdevjs) [inline Casks/touche.rb]: Done. 
For the future, where can I find an appcast url? is there any tool/cli for this?

Comment (vitorgalvao) [inline Casks/touche.rb]: @artdevjs [`appcast` docs](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md).

**Review by vitorgalvao — APPROVED** [review]

File: Casks/touche.rb (hunk 1 above)
Comment (artdevjs) [inline Casks/touche.rb]: already found the right section of the documentation

Comment (artdevjs) [inline Casks/touche.rb]: @vitorgalvao thx

**Pull request merged**